eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -157,7 +155,9 @@ public void onError(String errorMessage) {
                 })
                 .buildAndInit();
 
-        lock.await(2000, TimeUnit.MILLISECONDS);
+        if(!lock.await(10000, TimeUnit.MILLISECONDS)) {
+            throw new RuntimeException("Request for RAC did not complete within timeout");
+        }
```

Comment:
This is the core change. If [`CountDownLatch.await()`](https://developer.android.com/reference/java/util/concurrent/CountDownLatch#await(long,%20java.util.concurrent.TimeUnit)) hits it's timeout, it returns false but then continues execution. This can be confusing to the developer, so now we'll throw an error.

First testing this on my bad home wifi, I hit the error 😅 . So, I've extended the timeout to a generous 10 seconds. However, since it's using this await, most of the time we won't actually wait that long.